nexus/src/external_api/console_api.rs

```diff
@@ -67,7 +68,11 @@ pub async fn spoof_login(
 
     let user_id = user_id.unwrap();
 
-    let session = nexus.session_create(user_id).await?;
+    // For now, we use the external authn context to create the session.
+    // Once we have real SAML login, maybe we can cons up a real OpContext for
+    // this user and use their own privileges to create the session.
```

Comment:
This is interesting — we will think of the user as having the ability to create a session for themselves? I suppose in theory that would work for deletes in the logout case where they're logged in, but for delete in the expiration case they are no longer authenticated at delete time. Just thinking out loud.

Reply:
Yeah, that's fair.  I agree we wouldn't use this for logout.  I'm also not positive this will work when we flesh out SAML login -- it relies on us having created an OpContext for the user that's authenticated not by the session cookie but by the SAML assertions.  If that falls out of that work, I figured it'd be nice to use.  If it doesn't, I think it's totally fine to continue using this context.